.github/TEST_FAILURE_REPORT_TEMPLATE.md

```diff
@@ -5,4 +5,4 @@ labels: ['type::bug', 'type::testing', 'source::auto']
 
 The {{ workflow }} workflow failed on {{ date | date("YYYY-MM-DD HH:mm") }} UTC
 
-Full run: {{ repo.html_url }}/actions/runs/{{ env.RUN_ID }}
+Full run: https://github.com/{{ repo.owner }}/{{ repo.repo }}/actions/runs/{{ env.RUN_ID }}
```

Comment:
Fix for this incorrect URL:

<img width="975" alt="Screenshot 2025-03-06 at 14 39 51" src="https://github.com/user-attachments/assets/326cd57d-73c5-43ab-9ae1-9baee150236f" />